PayItForward.Web/Controllers/HomeController.cs

```diff
@@ -50,9 +60,30 @@ public IActionResult Index(int id, string categoryName, string search = "")
             return this.View(resultModel);
         }
 
-        public IActionResult About()
+        [Authorize]
+        public IActionResult MyStories()
         {
-            return this.View();
+            var stories = this.storiesService.Stories()
+                .Where(story => story.User.Id == this.httpaccessor.HttpContext.User.FindFirst(ClaimTypes.NameIdentifier).Value)
+                .ToList();
+
+            var resultModel = new MyStoriesViewModel()
+            {
+                MyStories = this.mapper.Map<List<BasicStoryViewModel>>(stories)
+            };
+
+            if (resultModel.MyStories.Count() < 1)
+            {
+                resultModel = new MyStoriesViewModel()
+                {
```

Comment:
You dont need this model about stories view model, and the message can be set in the view, depending on the action how many stories gives you back, so actually everything you need here is to map the stories to the  basicStoryview model list and return it with the view and after that the view can decide what is the message depending on the count of the stories